app/dts/behaviors/modifiers.dtsi

```diff
@@ -0,0 +1,94 @@
+#include <dt-bindings/zmk/keys.h>
+
+/ {
+	behaviors {
+		ctl: behavior_ctl {
+			compatible = "zmk,behavior-key-press";
+			label = "MOD_CTRL";
+			usage_page = <USAGE_KEYPAD>;
+			#binding-cells = <1>;
+            modifiers = <(MOD_LCTL)>;
```

Comment:
Used spaces here and a few other lines in the file while the rest of the file uses tabs for indentation.